Userland/Libraries/LibWeb/SVG/SVGScriptElement.cpp

```diff
@@ -19,4 +20,40 @@ void SVGScriptElement::initialize(JS::Realm& realm)
     set_prototype(&Bindings::ensure_web_prototype<Bindings::SVGScriptElementPrototype>(realm, "SVGScriptElement"));
 }
 
+void SVGScriptElement::visit_edges(Cell::Visitor& visitor)
+{
+    Base::visit_edges(visitor);
+    visitor.visit(m_script);
```

Comment:
I don't actually know for sure whether we need to hold on to this or not?

Reply:
I'm not 100% sure but I think we do need to keep it around. Otherwise, we could end up with an SVGScriptElement that is marked as "already processed" but has a null script.